sdcm/remote/libssh2_client/__init__.py

```diff
@@ -164,6 +160,10 @@ def stop_and_wait(self, timeout: float = None):
         self.stop()
         self.join(timeout)
 
+    def __del__(self):
+        self.stop_and_wait(1)
+        self._keep_running = None
```

Comment:
This reference will be destroyed automatically. Don't need to assign `None` explicitly.